src/components/Landing.jsx

```diff
@@ -1,38 +1,41 @@
-import { Button, Card, Col, Image, Row } from "react-bootstrap";
-import { Link } from "react-router-dom";
+import { Button, Card, Col, Image, Row } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
 
 const LandingPage = () => {
-  console.log("Here we go again!");
+  console.log('Here we go again!');
   return (
     <div
-      className="d-flex container"
+      // Here use .container-fluid instead of .container because .container has one fixed width for each screen size in bootstrap (xs,sm,md,lg,xl,xxl); .container-fluid expands to fill the available width.
+      className='d-flex container-fluid'
       style={{
-        minHeight: "100vh",
-        justifyContent: "center",
+        minHeight: '100vh',
+        justifyContent: 'center',
       }}
     >
       <Row
         style={{
-          alignItems: "center",
+          alignItems: 'center',
         }}
       >
-        <Col className="justify-content-center col-md-6 col-sm-10">
-          <Card className="border-0">
+        {/* Bootstrap includes different breaking points (xs, sm, md, lg, xl, xxl). For mobile responsiveness you have to use 'xs'.*/}
+        {/* Bootstrap breakpoints: https://react-bootstrap.github.io/layout/breakpoints/ */}
+        <Col className='justify-content-center col-md-6 col-sm-10 col-10'>
```

Comment:
Here you haven't used col-xs-*, yet how is it functioning well?
Is it considering col-10 as specifications for col-xs- since, col-md and col-sm are already specified?

Also same thing over here: [Line 33](https://github.com/Tuhin-thinks/ExpenseTraker/pull/1/files#diff-4c017db8e17fc8f6e4c0b8f57104d9a9b1df071165bd27f6a7bd3f79695f5a9fR33)

Reply:
```col-xs-*``` have been dropped in Bootstrap 4 in favor of ```col-*```. So instead of col-xs-10 you can simply write col-10